docs/utilities.rst

```diff
@@ -212,6 +212,50 @@ An example using the model specified above:
     {'title': None}
 
 
+Tracking Foreign Key Fields
+---------------------------
+
+It should be noted that a generic FieldTracker tracks Foreign Keys by db_column name, rather than model field name, and would be accessed as follows:
+
+.. code-block:: python
+
+    from django.db import models
+    from model_utils import FieldTracker
+
+    class Parent(models.Model):
+        name = models.CharField(max_length=64)
+
+    class Child(models.Model):
+        name = models.CharField(max_length=64)
+        parent = models.ForeignKey(Parent)
+
+.. code-block:: pycon
```

Comment:
Nah you're quite right, I'd entirely forgotten it was a separate lexer. In my head it was just a typo.